src/iplant_groups/clients/grouper.clj

```diff
@@ -14,10 +15,10 @@
 (def ^:private default-act-as-subject-id "GrouperSystem")
 
 (def ^:private all-group-privileges
-  ["view" "read" "update" "admin" "optin" "optout" "groupAttrRead" "groupAttrUpdate"])
+  #{"view" "read" "update" "admin" "optin" "optout" "groupAttrRead" "groupAttrUpdate"})
 
 (def ^:private all-folder-privileges
-  ["create" "stem" "stemAttrRead" "stemAttrUpdate"])
+  #{"create" "stem" "stemAttrRead" "stemAttrUpdate"})
```

Comment:
I changed the lists to set literals so that I could use `clojure.set/difference` to get the difference between two sets.